tests/integration/targets/mso_schema_site_anp_epg_staticport/tasks/main.yml

```diff
@@ -23,6 +23,13 @@
       use_proxy: '{{ mso_use_proxy | default(true) }}'
       output_level: '{{ mso_output_level | default("info") }}'
 
+# QUERY VERSION
+- name: Query MSO version
+  cisco.mso.mso_version:
+    <<: *mso_info
+    state: query
+  register: version
+
```

Comment:
Why adding version?

Reply:
Please explain why we need it.

Reply:
I added by mistake, reverted the change.